bazel/deps.bzl

```diff
@@ -21,7 +21,7 @@ def PI_deps():
             name = "com_github_p4lang_p4runtime",
             # urls = ["https://github.com/p4lang/p4runtime/archive/v%s.zip" % P4RUNTIME_TAG],
             urls = ["https://github.com/p4lang/p4runtime/archive/%s.zip" % P4RUNTIME_COMMIT],
-            sha256 = P4RUNTIME_SHA,
+            # sha256 = P4RUNTIME_SHA,
```

Comment:
will commenting this out always give a warning during bazel builds?
I think if you omit it and build locally, Bazel will give you the correct sha to use

Reply:
Fixed.